packages/uicore/src/components/ScrollShadowContainer/ScrollShadowContainer.module.css

```diff
@@ -0,0 +1,39 @@
+.body {
+  --shadowTop: inset 0px 8px 5px -5px rgba(0, 0, 0, 0.7);
+  --shadowBottom: inset 0px -8px 5px -5px rgba(0, 0, 0, 0.7);
+  position: relative;
+  overflow: auto;
+}
+
+.body::before {
+  content: '';
+  position: absolute;
+  inset: 0;
+  pointer-events: none;
+  z-index: 2;
+}
+
+.body .bodyContent {
+  position: relative;
+  z-index: 1;
+  height: 100%;
+  overflow: scroll;
+  scroll-behavior: smooth;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+  &::-webkit-scrollbar {
+    display: none; /* Safari and Chrome */
```

Comment:
why are we hiding the native scrollbars? this is very bad for accessibility.

Reply:
removed css